packages/block-library/src/button/edit.js

```diff
@@ -133,10 +134,60 @@ function ButtonEdit( props ) {
 
 	const [ isEditingURL, setIsEditingURL ] = useState( false );
 	const isURLSet = !! url;
-	const opensInNewTab = linkTarget === NEW_TAB_TARGET;
-	const nofollow = !! rel?.includes( NOFOLLOW_REL );
+
 	const isLinkTag = 'a' === TagName;
 
+	// Defines how block attributes map to link value and vice versa.
+	const linkValueAttrsToDataMapping = {
+		url: 'url',
```

Comment:
I think that we made need to run this thorough `prependHTML` (as in the `getUpdatedLinkAttributes`) although I wonder if `LinkControl` does this itself now already 😕